### PR TITLE
Add "NEEDS AUTOUPGRADE LABEL" part in github lifecycle

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Add "NEEDS AUTOUPGRADE LABEL" part in github lifecycle
| Fixed ticket | -

### CODE REVIEW

File: content/maintainers-guide/summary-github-lifecycle.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ If one "Waiting for ..." label has been applied, the team in charge will process
 
 There are some automated bots running on GitHub that will help maintainers to label the Pull Requests. For example Prestonbot is able to extract the new wordings and add the "Waiting for wording" label. You can read more about them [here][how-pull-requests].
 
+### NEEDS AUTOUPGRADE LABEL
```
Comment (kpodemski) [inline content/maintainers-guide/summary-github-lifecycle.md:88]:
```suggestion
### Needs autoupgrade label
```

Comment (nicosomb) [inline content/maintainers-guide/summary-github-lifecycle.md:88]: WHY ARE YOU SHOUTING HERE? 😛

Comment (kpodemski) [reply content/maintainers-guide/summary-github-lifecycle.md:88]: YOU SEE THE LABEL YOU SEND THE PR 💥

Comment (M0rgan01) [reply content/maintainers-guide/summary-github-lifecycle.md:88]: CALM DOWN GUYS

**Review by nicosomb — APPROVED** [review]

**Pull request merged**